src/assets/translations/en/main.json

```diff
@@ -994,7 +994,7 @@
     "allowanceApprovalTxName": "Allowance approval tx:",
     "tradeComplete": "You now have %{cryptoAmountFormatted} in your wallet on %{chainName}.",
     "doAnotherTrade": "Do another trade",
-    "showDetails": "Show Details",
+    "summary": "Trade Summary",
```

Comment:
@NeOMakinG iirc from @firebomb1, only english is required to be removed and translations from other languages are then removed next biweekly translations sync  from MKBD ws.

Reply:
Oh you're right, I thought neo only changed the key. Yeah we'll need to remove all non-english instances to have it re-translated.

Reply:
Will let firebomb step in since a key change is slightly different than a proper removal, but I believe that it should only show as a line changed for en, and as an *addition* for all others (keeping the old key/line in to be removed)

Reply:
So we should remove all other translations than the english one?

Reply:
@NeOMakinG Since you also change the value (the English copy) for this key after renaming it, you want translators to translate it again. In this case you can either remove them or you can simply leave the old translations associated with the old key name and they are going to be discarded the next time we do a translations sync (we only keep keys that exist in the English file).

If you were only renaming the key, updating the key name in all the translation files would makes sense because this way you'd preserve translations and spare us the cost of re-translating them for nothing... but that's not the case here, we want to translate them again.

Reply:
I reverted the keys then so they are going to be stale and discarded

Thanks!